input/resources/au-core-smokingstatus.xml

```diff
@@ -198,8 +198,8 @@
         <key value="au-core-obs-01"/>
         <severity value="error"/>
         <human value="Date shall be at least to day"/>
-        <expression value="$this is DateTime implies $this.toString().length() &gt;= 10"/>
-        <xpath value="f:matches(effectiveDateTime,/\d{4}-[01]\d-[0-3]\dT[0-2]\d:[0-5]\d([+-][0-2]\d:[0-5]\d|Z)/)"/>
+        <expression value="$this is DateTime implies $this.toString().length() &gt;= 10 or extension('http://hl7.org/fhir/StructureDefinition/data-absent-reason').exists()"/>
```

Comment:
hold off on this change - we would want to be consistent across all Observations on how to support Missing and Suppressed Data. This change shouldn't be made in isolation and requires the research and resolution of how to approach this on the Vitals.

Reply:
Done

Reply:
Update the JIRA that this is on hold.